src/index.css

```diff
@@ -1,13 +1,156 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap");
+
 body {
+  background-color: rgb(208, 67, 121);
+}
+
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  color: white;
+  font-family: "Open Sans", sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+}
+
+.details-header {
+  background-color: rgb(211, 70, 124);
+  display: flex;
+  align-items: center;
+  padding: 0.6rem 0.4rem;
+}
+
+.details-header p {
+  margin: 0 auto;
+}
+
```

Comment:
- [ ] It would be better if the image does not overflow the content. kindly fix this minor issue. SOme logos overflow the page🔧 😄 

![Capture](https://user-images.githubusercontent.com/64173189/148486511-32e60bc7-e067-4120-9cd7-bbb8ced20675.JPG)